webpack/webpack.config.dev.js

```diff
@@ -53,6 +53,8 @@ export default {
     new webpack.HotModuleReplacementPlugin(),
     new webpack.NoEmitOnErrorsPlugin(),
     new webpack.DefinePlugin({
+      __AUTH0_CLIENT_ID__: JSON.stringify(process.env.AUTH0_CLIENT_ID),
```

Comment:
this will bork in prod since the client is determined by the env, not the build.